generate_outputs.py

```diff
@@ -53,6 +53,9 @@
     parser.add_argument("production_csv_output_path", metavar="production-csv-output-path",
                         help="Path to a CSV file to write raw message and demographic responses to, for use in "
                              "radio show production"),
+    parser.add_argument("auto_coding_json_output_path", metavar="auto-coding-json-output-path",
+                        help="Path to a JSON file to write the TracedData associated with auto-coding stage of the pipeline"),
+    parser.add_argument("generate_analysis_files", help="determines if to generate analysis files or not. Set to true/false")
```

Comment:
```suggestion
    parser.add_argument("generate_analysis_files", help="whether to generate analysis files. Set to true/false")
```
It's also possible to have argparse do the true/false check for you:
```suggestion
    parser.add_argument("generate_analysis_files", help="determines if to generate analysis files or not", choices=["true", "false"])
```